src/metabase/api/table.clj

```diff
@@ -132,7 +133,7 @@
                      :type "type/DateTime"})
                   ;; note the order of these options corresponds to the order they will be shown to the user in the UI
                   [[(deferred-tru "Minute") "minute"]
-                   [(deferred-tru "Hour") "hour"]
+                   [hour-str "hour"]
```

Comment:
👍 on doing this for `Hour` but why not do it for `Minute` also

Reply:
Good catch, thanks! I've added that.